superset/views/core.py

```diff
@@ -1727,7 +1727,8 @@ def testconn(self):
 
             with closing(engine.connect()) as conn:
                 conn.scalar(select([1]))
-                return json_success('"OK"')
+                return json_success(json.dumps(engine.table_names(), indent=4))
```

Comment:
It's intentional that we don't dump the table names when testing the connection. This api shouldn't be used for anything more than making sure the connection to a db works

Reply:
Yes that's what it did historically for no really good reason (returning table names). Returning a 200 "OK"seems right.